dap_job_quality/notebooks/cluster_sentences.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
Because its an .ipynb file, i've just added some notes here:

`cluster_sentences.ipynb`:
- I needed to install matplotlib and spacy to load imports;
- I needed to add `job_quality` to your paths when you `load_s3_jsonl`
- I would save the labelled data locally to the `outputs` file instead 
- agree - i don't think there's a ton to say about the clusters but a good investigation into labels nonetheless!
- everything else ran!